src/sentry/api/serializers/models/release.py

```diff
@@ -59,10 +59,21 @@ def get_users_for_commits(item_list):
 
     author_objs = {}
     for author in authors:
-        author_objs[author.id] = users_by_email.get(author.email, {
-            "name": author.name,
-            "email": author.email
-        })
+        if users_by_email.get(author.email):
```

Comment:
does this actually impact the number of queries we're running? generally, trying to either remove or optimize queries is going to have the best chance of improving performance